packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsForm/PayableDetailsForm.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import { FC, useState } from 'react';
+
+import { Dialog } from '@/components/Dialog';
+import { action } from '@storybook/addon-actions';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import {
+  PayableDetailsForm,
+  PayableDetailsFormProps,
+} from './PayableDetailsForm';
+
+const meta: Meta<typeof PayableDetailsForm> = {
+  title: 'Payables/Payables — Details Form',
+  component: PayableDetailsForm,
+};
+
+const payableId = '9a3b97a5-a1ba-4d8c-bade-ad3c47ae61e0';
+
+type Story = StoryObj<typeof PayableDetailsForm>;
+
+const actions = {
+  onSavePayable: action('onSavePayable'),
+  onCreatePayable: action('onCreatePayable'),
+};
+
+const WithDialogComponent: FC<PayableDetailsFormProps> = (args) => {
+  const [open, setOpen] = useState(true);
+
+  return (
+    <Dialog
+      open={open}
+      fullScreen
+      onClose={() => {
+        action('onClose')();
+        setOpen(false);
+      }}
+      onSubmit={() => {}}
+      onClosed={action('onClosed')}
+    >
+      <PayableDetailsForm {...args} />
```

Comment:
The story implementation does not show how could we configure these fields.

Reply:
Are you referring to the OCR fields? So, do I need to set up the entire story with MUI extended support just for these specific fields? I’m concerned because we’re on a tight deadline, and spending time on something that may not be necessary could slow us down. Are you sure this is critical and important?

Reply:
I added a corresponding story